nexus/client.go

```diff
@@ -229,12 +239,21 @@ func (c *Client) StartOperation(ctx context.Context, operation string, input any
 		if info.State != OperationStateRunning {
 			return nil, newUnexpectedResponseError(fmt.Sprintf("invalid operation state in response info: %q", info.State), response, body)
 		}
+		links, err := getLinksFromHeader(response.Header)
+		if err != nil {
+			return nil, newUnexpectedResponseError(
```

Comment:
Maybe wrap with `fmt.Errorf("%w: %w", newUnexpectedResponseError(...), err)` to avoid losing the original error.

Reply:
I wonder if we should return error at all since this is the case the response returned status CREATED, and it's just the links header that was unable to be decoded.